runtime/plugin/tutor.vim

```diff
@@ -0,0 +1,6 @@
+if exists('g:loaded_tutor_mode_plugin') || &compatible
```

Comment:
No need for ` || &compatible`, unless you're trying to reduce the diff between the vim-tutor-mode repo.